notebooks/gpu-footprint.ipynb

```diff
@@ -0,0 +1,297 @@
+{
```

Comment:
<p>These bigger models do not load, right?</p><p> </p><p>It might be worth adding a short note about that - maybe it would not hurt to also include the output with the error</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/35/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>This results in OOM but yeah I'll add a note.</p>